pkg/build/openbsd.go

```diff
@@ -71,15 +71,15 @@ func (ctx openbsd) make(kernelDir string, args ...string) error {
 	return err
 }
 
-// CopyFilesToImage populates the filesystem image in outputDir with
+// copyFilesToImage populates the filesystem image in outputDir with
 // run-specific files. The kernel is copied as /bsd and if overlayDir
 // exists, its contents are copied into corresponding files in the
 // image.
 //
 // Ideally a user space tool capable of understanding FFS should
 // interpret FFS inside the image file, but vnd(4) device would do in
 // a pinch.
-func CopyFilesToImage(overlayDir, outputDir string) error {
+func copyFilesToImage(overlayDir, outputDir string) error {
```

Comment:
It's probably better to also prefix them with OS name, or make method on the ctx for the purposes of name scoping.